plugins/chat/assets/javascripts/discourse/controllers/chat.js

```diff
@@ -23,6 +24,17 @@ export default class ChatController extends Controller {
     return this.siteSettings.navigation_menu === "sidebar";
   }
 
+  get activeTab() {
+    switch (this.router.currentRouteName) {
+      case "chat.threads":
+        return "threads";
+      case "chat.direct-messages":
+        return "direct-messages";
+      case "chat.channels":
+        return "channels";
+    }
```

Comment:
How about this one-liner?

```suggestion
    return this.router.currentRouteName.replace(/^chat\./, "");
```